backoffice/backoffice-domain-service/ballerina/backoffice-api_service.bal

```diff
@@ -62,12 +62,17 @@ service /api/am/backoffice on ep0 {
         }
         return apiDefinition;
     }
+
+    isolated resource function get apis/[string apiId]/thumbnail(@http:Header string? 'if\-none\-match, @http:Header string? accept = "application/json") returns http:Response|commons:APKError {
```

Comment:
shall we add them to the same level as previous commented out methods, otherwise with open-api bal gen command again these methods location will change

Reply:
Fixed